salt/transport/tcp.py

```diff
@@ -425,7 +611,7 @@ def handle_stream(  # pylint: disable=arguments-differ
         unpacker = salt.utils.msgpack.Unpacker()
         try:
             while True:
-                wire_bytes = yield stream.read_bytes(4096, partial=True)
+                wire_bytes = await stream.read_bytes(4096, partial=True)
```

Comment:
Noticed the 4k earlier too, but why 4k, ubiquitous 4k, but NFS has an 8k block (Digital era), hence the 912x large frame on Ethernet. Just wondering justification for 4k.

Reply:
We're going with what the value has been since originally written. Unless there is a compelling need to change, I'd prefer not too unless it's extremely well tested. That said, it falls outside of the scope of this PR.